test/task_serialize_test.py

```diff
@@ -0,0 +1,113 @@
+"""
+We want to test that task_id is consistent when generated form:
```

Comment:
generated *from*

Legally I suppose this file needs one of those license header thingies. :)